eth/catalyst/api.go

```diff
@@ -334,7 +347,16 @@ func (api *ConsensusAPI) ExchangeTransitionConfigurationV1(config beacon.Transit
 }
 
 // GetPayloadV1 returns a cached payload by id.
-func (api *ConsensusAPI) GetPayloadV1(payloadID beacon.PayloadID) (*beacon.ExecutableDataV1, error) {
+func (api *ConsensusAPI) GetPayloadV1(payloadID beacon.PayloadID) (*beacon.ExecutableData, error) {
```

Comment:
Nitpick, I would propose to define an internal function `getPayload`, 

then define 

```
func GetPayloadV1(id) (*beacon.ExecutableData, error) {
     payload := api.getPayload(id)
     return payload.ExecutionPayload, nil
}

func GetPayloadV2(id) (*beacon.ExecutableData, error) {
     payload := api.getPayload(id)
     return payload, nil
}
```

It's a bit weird to call v2 logic in v1; feels more natural to call v1 logic in v2.

But anyway, totally fine to ignore this nitpick comment.